site/_posts/2017-01-06-sprint-51-review-january-4-2017.html.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Sprint 51 review - January 4, 2017
+author: cybette
+date: 2017-01-06 11:15:19 UTC
+tags: sprints
+comments: true
+published: true
+---
+
+Welcome to the first sprint review of 2017! It was a relatively succinct meeting, so I'll try to keep this short and sweet. First, a couple of news items in case you missed them:
+
+* Just before the holidays, we released [ManageIQ Euwe GA](/blog/2016/12/manageiq-euwe-ga-announcement/). Onwards to [Fine](/blog/2016/11/announce_f_name/)!
+* Length of ManageIQ Sprints change from [3 weeks to 2 weeks](/blog/2017/01/schedule-change-3-week-sprints-to-2-week-sprints/), with immediate effect.
+
+During this 4-week sprint filled with holiday spirit, we sprinkled in 249 pull requests in the [main repo](https://github.com/ManageIQ/manageiq) and 439 PRs [overall](https://github.com/manageiq). While the total number is smaller than usual, there are more enhancement PRs than the previous sprint. But before we touch on some of these enhancements, here's an important change in the ManageIQ repository distribution.
+
+Major effort went into the creation of 4 new repos (splitting them from the main repo): Classic UI, Content (Automate Domain), PGLogical Replication, and Design. The one with the biggest impact is Classic UI, which frequently took the largest slice of the main repo PR pie. If you work on UI, be sure to read the [discussion on the talk forum](http://talk.manageiq.org/t/new-split-repo-manageiq-ui-classic/1983).
+
+Other repos on the splitting block for this year include Database Schema, Automate Engine, REST API, and each of the Providers. This is the first phase of transforming ManageIQ from a monolithic app to a platform, with each module having its own test suites and services, ensuring code coverage and quality throughout development. Discuss and stay informed about the progress on [ManageIQ Talk](http://talk.manageiq.org/).
+
+In Classic UI, columns displaying virtual CPU and memory values for Cloud Providers are added, along with a new menu item to provision a Floating IP in Network. Notifications of asynchronous operations are now shown for Middleware. Several screens have been moved from Classic UI to Service UI, such as approval requests for admins, view service / VM details, and list / create / remove service catalogs.
+
+New Chargeback features include support for fixed and allocated costs without metrics, and arbitration logic for multiple assigned rates. On the Automate side, enhanced messaging has been extended to Ansible and Cloud Provisioning, and importing Automate modeling from the UI now honors the tenant.
+
+For VMware vCloud, XLAB added the ability to specify VM attributes when deploying vApps. Another big step for providers in general is the new Provider Generator, which was [proposed](https://www.youtube.com/watch?v=i5ZuTKFQqJ8) in last year's Design Summit. With a single line of command, `rails generate provider ProviderName` will generate the scaffolding for a provider. 
```

Comment:
I feel like these are two separate bullet items that shouldn't be together.  Initially I read the "Another big step" as being related to the XLAB work, but it's not.